docs/language/src/concepts/translations.md

```diff
@@ -46,8 +58,93 @@ export component MenuItem {
 
 ## Extracting the String from the files
 
-**TODO**
+We provide a `slint-tr-extractor` tool that parse .slint files and can generate a .po file.
+You can run it like so
+
+```sh
+find -name \*.slint | xargs slint-tr-extractor -o MY_PROJECT.pot
+```
+
+This will create a file called `MY_PROJECT.pot`
+(but you should replace MY_PROJECT with your actual project name, see bellow for naming)
 
 ## Doing the translations
 
-**TODO**
\ No newline at end of file
+`.pot` file are [Gettext](https://www.gnu.org/software/gettext/) template files. It is the same as a `.po` file, but doesn't contain actual
+translations. They can be edited by hand with a text editor, or there are a few tools you can use to translate them, option includes
+ - [poedit](https://poedit.net/)
+ - [OmegaT](https://omegat.org/)
+ - [Lokalize](https://userbase.kde.org/Lokalize)
+ - [Transifex](https://www.transifex.com/) (web interface)
+
+## Loading the translations at runtime
+
+[Gettext](https://www.gnu.org/software/gettext/) is used at runtime to get the translations.
```

Comment:
"get" is a bit unspecific here. I suggest to remove this sentence in favor of the aforementioned translation.